modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -120,21 +94,34 @@ xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read
 * Xen
 * KVM
 
-=== Desktop
+=== Mobile OS
```

Comment:
Why did this heading change to "Mobile OS"?
The items under here look like Desktop OS.

Reply:
Not sure how that slipped through. Will correct.